### ISSUE TITLE
Using pre-trained weights for LSUN Horses from e4e.

### ISSUE THREAD

**Issue by arturandre** [eladrich/pixel2style2pixel#286]
Hi, I'm working with the LSUN Horses dataset and I noticed that in the [ReStyle repo]([https://github.com/yuval-alaluf/restyle-encoder](https://github.com/yuval-alaluf/restyle-encoder/)) there are pre-trained weights for the e4e encoder. Is there a straightforward way to use the e4e encoder from ReStyle repository to perform sketch-to-image translation in this repository? Thank you.

**Comment by yuval-alaluf** [comment]
Sure. What I would do is add support for the sketch-to-image dataset from this repo into the ReStyle repo. Other than changing the dataset, there shouldn't be any other changes needed (other than maybe adjusting the loss weights). You can take a look in this repo of how to define the sketch-to-image dataset and add that to the ReStyle code.

**Comment by arturandre** [comment]
Thank you very much! I'll look into that.
